bin/build.mjs

```diff
@@ -55,16 +55,46 @@ async function buildPackage(absWorkingDir) {
    * @param {'esm' | 'cjs'} format
    */
   const entryPoints = (format) => {
-    let outFilename = filename + (format === 'esm' ? '.module' : '');
-    outFilename += DEV ? '' : '.min';
+    /**
+     * @param {string} base
+     */
+    const addSuffix = (base) => {
+      let outFilename = base + (format === 'esm' ? '.module' : '');
+      return outFilename + (DEV ? '' : '.min');
+    };
+    let outFilename = addSuffix(filename);
+
+    const extraEntryPoints = ['@pixiv/three-vrm', '@pixiv/three-vrm-materials-mtoon'].includes(packageJson.name)
```

Comment:
doing this here to modify cjs,esm outputs altogether

Reply:
This section looks surprisingly cryptic compared to what it actually does 😩